README.md

```diff
@@ -34,7 +34,7 @@ To create a template:
 ```shell
 📁 my_copier_template ------------------------ # your template project
```

Comment:
Why not here?

Reply:
It's the README, it wouldn't be rendered on GitHub.